examples/website/i3s/app.js

```diff
@@ -109,16 +112,25 @@ export default class App extends PureComponent {
     this._onSelectTileset(tileset);
   }
 
-   /**
+  /**
    * Tries to get Building Scene Layer sublayer urls if exists.
-   * @param {string} tilesetUrl 
+   * @param {string} tilesetUrl
    * @returns {string[]} Sublayer urls or tileset url.
    * TODO Add filtration mode for sublayers which were selected by user.
    */
   async getLayerUrls(tilesetUrl) {
     try {
       const tileset = await load(tilesetUrl, I3SBuildingSceneLayerLoader);
-      return tileset?.sublayers.map(sublayer => sublayer.url);
+      const sublayersTree = buildSublayersTree(tileset.header.sublayers);
+      this.setState({sublayers: sublayersTree.sublayers});
+      const urls = tileset?.sublayers
+        .map((sublayer) => sublayer.url)
+        .filter(
+          (url) =>
+            url !==
+            'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/Admin_Building_v17/SceneServer/layers/0/sublayers/0'
```

Comment:
I am not sure that we should save this line.

Reply:
We should. It is "Preview" model. We shouldn't show "Preview" and "Full Model" at the same time